### ISSUE TITLE
curl: (7) Failed connect to raw.githubusercontent.com:443; （A suggestion to change the README）

### ISSUE THREAD

**Issue by WL2O2O** [GreptimeTeam/dashboard#432]
![image](https://github.com/GreptimeTeam/dashboard/assets/72431275/bc60390c-84ba-41e8-bd53-e4f8c0b632fc)
When I pull the `docker-compose` configuration file in a local VM or cloud server using the approach suggested in the above figure, I encountered a problem with the network environment, i.e., accessing the web page [https://raw.githubusercontent.com](https://raw. githubusercontent.com), was rejected with the error message "curl: (7) Failed connect to raw.githubusercontent.com:443; Connection refused". It's pretty easy to think of what's causing this little problem, so I'd suggest modifying the `README` File as a way to minimize the number of times others have to step in the hole.

Regarding changes to the `README` file:
### With Docker
⭐We recommend using docker-compose to setup both greptimedb and dashboard.
1. Use the `curl` command to get the `docker-compose` configuration file
> Tips: If you are located in China and do not have access to an external network, you are likely to encounter network-related problems when executing the following commands.
```
curl -O https://raw.githubusercontent.com/greptimeteam/dashboard/main/docker/docker-compose.yml
```
> Solution: Download the `docker-compose.yml` file locally and transfer it to the VM or server using `SFTP`. Or just copy the contents of [this file](https://github.com/GreptimeTeam/dashboard/blob/main/docker/docker-compose.yml) and create the file in the virtual machine or server.
2. Launching the application
```
docker compose up
```
3. Open you browser and visit http://localhost:8080/dashboard/
- Follow getting started to create your table and insert some data
- Run a sELECT query and check the result as table or chart

Best regards to Greptime Team!

**Comment by killme2008** [comment]
Thanks for your detailed report.

GreptimeDB already embeds the dashboard, you can find it at `http://{greptimedb_host}:4000/dashboard`.

**Comment by WL2O2O** [comment]
> Thanks for your detailed report.
> 
> GreptimeDB already embeds the dashboard, you can find it at `http://{greptimedb_host}:4000/dashboard`.

@killme2008 You're welcome, it's inevitably good news. 

But I only meant to refine the `README` documentation for the repository dashboard. I saw a need to raise an `issue` about this repository and address it in a front-end internship post by @alili, but that post was posted 6 months ago, ahaha, This is why I called `GreptimeDB already embeds the dashboard` a necessarily good news. But due to my interest in `GreptimeDB`, I raised a `issue` that probably wasn't necessary anyway. 

Thank you for your reply, best Wishes!
